src/core/server/elasticsearch/legacy/scoped_cluster_client.ts

```diff
@@ -66,12 +66,12 @@ export class LegacyScopedClusterClient implements ILegacyScopedClusterClient {
     clientParams: Record<string, any> = {},
     options?: LegacyCallAPIOptions
   ) {
-    if (this.auditor) {
-      this.auditor.add({
-        message: endpoint,
-        type: 'elasticsearch.call.internalUser',
-      });
-    }
+    // if (this.auditor) {
```

Comment:
Don't forget to uncomment before merging 😄

Reply:
Haha, indeed

Reply:
We discussed offline today, and we now think it makes sense to remove these calls altogether from the cluster client, as we don't have a way to determine the high level actions at this point. For example, is the request a `create`, `read`, or something entirely different? We have no way to categorize this information, so it feels like keeping these calls in would just add unhelpful noise to the logs.

Phase 1 does not need to capture all audit events, so removing this now won't cause us to lose anything we care about in the short term.

Reply:
As discussed, I will remove these for now

Reply:
Are you going to put them back later? Otherwise, it doesn't make sense to have the `audit trail` service in the platform. Security plugin already wraps all the SO client methods.

Reply:
This has been removed from core now